backend/experiment/static/experiment_admin.css

```diff
@@ -0,0 +1,24 @@
+.buttons-row {
+	margin: 0px 0px 0px 10px;
+}
+
+.buttons-row .button {
+	padding: 2px 5px;
+	margin: 0px 0px 0px 5px;
+}
+
+.buttons-column {
+	margin: 10px 0px 0px 0px;
+}
+
+.buttons-column button {
+	display: block;
+	padding: 2px 5px;
+	margin: 5px 0px 0px 0px;
+}
+
+form .button-show-hide {
+ 	width: 45px;
+	padding: 2px 5px;
+	margin: 0px 10px 0px 0px;
```

Comment:
Instead of absolute pixel values, better use relative values such as vh / vw. While we would expect most users not to use the admin interface with mobile devices, during field work, it would be nice if the admin interface scaled.

Reply:
We discussed, Django Admin actually uses absolute values all over, so it may actually lead to unexpected results to use relative values for this custom addtion.